CMakeLists.txt

```diff
@@ -160,7 +160,7 @@ if(WIN32)
 	endif(MSVC)
 
 	if(MINGW)
-		set(SYSTEM_LIBS ${SYSTEM_LIBS} ole32 oleaut32 ws2_32 mswsock dbghelp)
+		set(SYSTEM_LIBS ${SYSTEM_LIBS} ole32 oleaut32 ws2_32 mswsock dbghelp bcrypt)
```

Comment:
Why this way? Can't we just move line 159 outside of the if statement?

Reply:
We can , but one shared line of code is not worth it.